lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py

```diff
@@ -147,12 +147,12 @@ def check_stack(self, process, pid, filename):
         self.check_backtrace(thread, filename, backtrace)
 
     @skipIfLLVMTargetMissing("AArch64")
-    def test_aarch64(self):
+    def test_aarch64_A(self):
```

Comment:
Can we give this a more meaningful name, like `test_aarch64_single` or `test_aarch64_single_threaded`? Or maybe keep these and only rename the ones below.

Reply:
Sure. I have renamed them to suggested names. Thanks.